src/v/kafka/server/handlers/list_offsets.cc

```diff
@@ -69,7 +69,7 @@ static ss::future<list_offset_partition_response> list_offsets_partition(
     auto kafka_partition = make_partition_proxy(ntp, mgr);
     if (!kafka_partition) {
         co_return list_offsets_response::make_partition(
-          ntp.tp.partition, error_code::unknown_topic_or_partition);
+          ntp.get_partition(), error_code::unknown_topic_or_partition);
```

Comment:
`get_partition()` is a member of `ktp` class (see `ktp.h`) and `ntp` here is of type `ktp`. 

When I changed the type from `model::ntp` to `model::ktp` I did not generally also change the variable name, even if it was `ntp`. I could though, do you think it's confusing as it is?

Reply:
Got it. Yes I actually find it confusing, since ktp is not an ntp or vice versa (in LSP sense), so calling `ntp()` and getting back a `ktp` feels weird.